packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1390,6 +1390,8 @@ class ListView extends BoxScrollView {
   /// `addSemanticIndexes` argument corresponds to the
   /// [SliverChildBuilderDelegate.addSemanticIndexes] property. None may be
   /// null.
+  /// The `includeSeparatorForLastItem` argument determines whether to add
+  /// a separator after the last item or not. Defaults to `False`.
```

Comment:
Hello, I didn't mean to interfere. But, can I suggest this?
```suggestion
  /// a separator after the last item or not. Defaults to `false`.
```

Reply:
Sure